src/pgduckdb.cpp

```diff
@@ -14,10 +14,16 @@ static void DuckdbInitGUC(void);
 
 bool duckdb_execution = false;
 int duckdb_max_threads_per_query = 1;
-char *duckdb_background_worker_db = NULL;
+char *duckdb_background_worker_db = strdup("postgres");
```

Comment:
I'm not sure if/how this actually works. It seems scary to allocate with `pstrdup` before `_PG_init` is necessarily called.

Reply:
I'm very surprised this passes CI.

Reply:
Okay I figured it out. `MemoryContextInit` is called in Postgres its `main` function before it does any of the `dlopen` calls for the `shared_preload_libraries`. So yeah I guess this is fine.

Reply:
Yeah, I noticed that C++ cares about const char vs non const char when I created that option. It's indeed reasonable. I'm fine with the strdup approach, now that I understand why it works, and I'm convinced that it will continue working.